server/routes/journeys/record-decision/conclusion/controller.ts

```diff
@@ -0,0 +1,39 @@
+import { Request, Response } from 'express'
+import { SchemaType } from './schemas'
+import { BaseJourneyController } from '../../base/controller'
+
+export class ConclusionController extends BaseJourneyController {
+  GET = async (req: Request, res: Response) => {
+    const outcomeTypeOptions = this.customRadios(
+      await this.getReferenceDataOptionsForRadios(
+        req,
+        'outcome-type',
+        res.locals.formResponses?.['outcome'] || req.journeyData.decisionAndActions!.outcome,
+      ),
+    )
+    res.render('record-decision/conclusion/view', {
+      conclusion: res.locals.formResponses?.['conclusion'] || req.journeyData.decisionAndActions?.conclusion,
+      outcomeTypeOptions,
+      backUrl: '../record-decision',
+    })
+  }
+
+  private customRadios = (items: { value: string; text: string | undefined }[]) => {
+    const nfaIndex = items.findIndex(o => o.value === 'NFA')
+    if (nfaIndex > -1) {
+      const nfa = items[nfaIndex]!
+      items.splice(nfaIndex, 1)
+      items.push(nfa)
+    }
+    const opeIndex = items.findIndex(o => o.value === 'OPE')
+    if (opeIndex > -1) {
+      items.splice(opeIndex, 1)
+    }
+    return items
+  }
```

Comment:
i think this is a repeat of what was in /screen ?
can this be an exported function to be shared by both controller?

Reply:
not entirely, we're also getting rid of one of the options here too